frontend/src/components/layouts/FixedHeightLayout.js

```diff
@@ -19,13 +22,44 @@ const useStyles = makeStyles({
   },
 });
 
-export default function FixedHeightLayout({ title, children }) {
+export default function FixedHeightLayout({ 
+  title, 
+  children,
+  message,
+  messageType
+}) {
   const classes = useStyles();
-
+  const [initialMessageType, setInitialMessageType] = React.useState(null);
+  const [alertOpen, setAlertOpen] = React.useState(true);
+  const [initialMessage, setInitialMessage] = React.useState("");
```

Comment:
Makes sense! I need to get used to this because I used to always leave the `React.` in there but I'll import the hook in the future and call it without `React.`